aoirint_hg100rclient/client.py

```diff
@@ -159,6 +172,33 @@ def get_wan_ipv4(
   ipv4_addr = result['NET_WAN_IPv4_Address']
   return ipv4_addr
 
+def get_wan_ipv6(
+  api_url: str,
+  token: str,
+  timeout: float = 3.0,
+) -> str:
```

Comment:
The type hint of the return value should be`typing.List[str]`.

(Use `typing.List[str]` instead of `list[str]` for Python 3.8 compatibility.)

Reply:
Thank you for your review.
Unfortunately I didn't install Python 3.8 or lower now, so I'll install it and fix it. please wait a moment.